src/analyze/BuildSystem.re

```diff
@@ -234,10 +244,11 @@ let isRunningInEsyNamedSandbox = () => {
 };
 
 let getExecutableInEsyPath = (exeName, ~pwd) => {
+  let whichCommand = Sys.win32 ? "where" : "which"
```

Comment:
wouldn't `where` work for *NIX systems too? perhaps we can take the opportunity to unify that here.

Reply:
`where` is not a command that is present by default on either my OSX or Manjaro Linux environment, so I don't think so.

Reply:
(The `esy` environment does have `which`, but it does the wrong thing for Windows - it gives a cygwin-style path like `/cygdrive/c/my/project...` instead of a windows-native path like `C:\my\project`)